deploy/000025-create-test-network.ts

```diff
@@ -1,49 +1,97 @@
 const BNT_VIRTUAL_BALANCE = 1;
 const BASE_TOKEN_VIRTUAL_BALANCE = 2;
 
-const INITIAL_DEPOSITS = {
-    [InstanceName.TestToken1]: toWei(50_000),
-    [InstanceName.TestToken2]: toWei(500_000),
-    [InstanceName.TestToken3]: toWei(1_000_000),
-    [InstanceName.TestToken4]: toWei(2_000_000),
-    [InstanceName.TestToken5]: toWei(3_000_000),
-    [InstanceName.TestToken6]: toWei(100_000, new TokenData(TokenSymbol.TKN6).decimals())
-};
-
 const TOKENS = [
-    { symbol: TokenSymbol.TKN1, instanceName: InstanceName.TestToken1 },
-    { symbol: TokenSymbol.TKN2, instanceName: InstanceName.TestToken2 },
-    { symbol: TokenSymbol.TKN3, instanceName: InstanceName.TestToken3 },
-    { symbol: TokenSymbol.TKN4, instanceName: InstanceName.TestToken4, tradingDisabled: true },
-    { symbol: TokenSymbol.TKN5, instanceName: InstanceName.TestToken5, depositingDisabled: true },
-    { symbol: TokenSymbol.TKN6, instanceName: InstanceName.TestToken6 }
+    {
+        symbol: TokenSymbol.TKN1,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken1,
+        initialDeposit: toWei(50_000),
+        depositLimit: toWei(5_000_000),
+        fundingLimit: toWei(10_000_000)
+    },
+    {
+        symbol: TokenSymbol.TKN2,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken2,
+        initialDeposit: toWei(500_000),
+        depositLimit: toWei(5_000_000),
+        fundingLimit: toWei(10_000_000)
+    },
+    {
+        symbol: TokenSymbol.TKN3,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken3,
+        initialDeposit: toWei(1_000_000),
+        depositLimit: toWei(5_000_000),
+        fundingLimit: toWei(10_000_000)
+    },
+    {
+        symbol: TokenSymbol.TKN4,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken4,
+        initialDeposit: toWei(2_000_000),
+        depositLimit: toWei(5_000_000),
+        fundingLimit: toWei(10_000_000),
+        tradingDisabled: true
+    },
+    {
+        symbol: TokenSymbol.TKN5,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken5,
+        initialDeposit: toWei(3_000_000),
+        depositLimit: toWei(5_000_000),
+        fundingLimit: toWei(10_000_000),
+        depositingDisabled: true
+    },
+    {
+        symbol: TokenSymbol.TKN6,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken6,
+        initialDeposit: toWei(100_000, new TokenData(TokenSymbol.TKN6).decimals()),
+        depositLimit: toWei(5_000_000),
+        fundingLimit: toWei(10_000_000)
+    },
+    {
+        symbol: TokenSymbol.TKN7,
+        initialSupply: toWei(1_000_000_000),
+        instanceName: InstanceName.TestToken7,
+        initialDeposit: toWei(100_000, new TokenData(TokenSymbol.TKN6).decimals()),
+        depositLimit: MAX_UINT256,
+        fundingLimit: MAX_UINT256
+    }
```

Comment:
Perhaps a utility function which returns each one of these objects.
Something like:
```
const token = (symbol, instanceName, initialSupply, initialDeposit, fundingLimit, depositingDisabled) => {
    const decimals = new TokenData(symbol).decimals();
    return {
        symbol,
        instanceName,
        initialSupply: toWei(initialSupply, decimals),
        initialDeposit: toWei(initialDeposit, decimals),
        depositLimit: toWei(depositLimit, decimals),
        fundingLimit: toWei(fundingLimit, decimals),
        depositingDisabled
    };
}
```

Reply:
It's a local test code, so we don't need to be that verbose here

Reply:
👍